src/collections/_documentation/accounts/quotas/index.md

```diff
@@ -0,0 +1,216 @@
+
+{% include components/alert.html
+    title="Note"
+    content="Delete and discard and per-project rate limits are available only for Business and Enterprise plans"
+    level="warning"
+%}
+
+## Increasing Quotas
+
+Add to your quota at any time during your billing period, either by upgrading to a higher tier or by increasing your on-demand capacity. While the available plans fit most individual and business needs, Sentry is designed to handle large throughput. If your team needs more, we’re happy to help. Reach out to our sales team at [sales@sentry.io](mailto:sales%40sentry.io) to learn more about increasing capacity.
+
+If you’ve exceeded your quota threshold, the server will respond with a 429 HTTP status code. However, if this is your first time exceeding quota, you'll be entered into a one-time grace period. For more information, see this [Help article](https://help.sentry.io/hc/en-us/articles/115000154554-What-happens-when-I-run-out-of-event-capacity-and-a-grace-period-is-triggered-). In addition, you can specify a spending cap for on-demand capacity if you need additional events; for example, if you’re rolling out a new version and anticipate more events this month. For more information, see [On-Demand Spending Cap]({%- link _documentation/accounts/pricing.md -%}#on-demand-cap).
+
+## Rate Limiting Projects {#id1}
+
+Per-key rate limits allow you to set the maximum volume of events a key will accept during a period of time.
+
+For example, you may have a project in production that generates a lot of noise. A rate limit allows you to set the maximum amount of data to “500 events per minute”. Additionally, you can create a second key for the same project for your staging environment, which is unlimited, ensuring your QA process is still untouched.
+
+To set up rate limits, navigate to **[Project] » Client Keys » Configure**. Select an individual key or create a new one, then you’ll be able to define a rate limit as well as view a breakdown of events received by that key. For additional information and examples, see [Rate Limiting in our guide to Manage Your Event Stream]({%- link _documentation/accounts/quotas/manage-event-stream-guide.md -%}#4-rate-limiting).
+
+{% include components/alert.html
+    title="Note"
+    content="Per-key rate limiting is available only for Business and Enterprise plans"
+    level="warning"
+%}
+
+## Inbound Filters {#inbound-data-filters}
+
+In some cases, the data you’re receiving in Sentry is hard to filter, or you don’t have the ability to update the SDK’s configuration to apply the filters. Sentry provides several methods to filter data server-side, which apply before checking for potential rate limits. Inbound filters include:
+
+-     Common browser extension errors
+-     Events coming from localhost
+-     Known legacy browsers errors
+-     Custom filters to filter out errors
+-     By their error message
+-     From specific release versions of your code
+-     From certain IP addresses.
```

Comment:
Web crawlers could be added to this list.

Reply:
Will add. Thank you.